src/app/lib/analyticsUtils/sendBeacon/index.test.js

```diff
@@ -80,6 +80,14 @@ describe('sendBeacon', () => {
         expect(reverbMock.viewEvent).toHaveBeenCalledTimes(1);
       });
 
+      it('should not call Reverb viewEvent if Reverb is not enabled for a service', async () => {
+        const sendBeacon = require('./index').default;
```

Comment:
Would it be better to declare the `sendBeacon` at a higher level within the describe block so that it can be re-used for each unit test or does it have to be declared each time?

Reply:
I've updated how the test is set up to enable us to mock `onClient` 
in a manner that enables use to import `sendBeacon` once.